backend/api/v2beta1/pipeline_spec.proto

```diff
@@ -35,11 +34,6 @@ message PipelineSpec {
   // Optional input field. The raw pipeline JSON spec.
```

Comment:
should we remove `string workflow_manifest = 2;` ?

Reply:
Yes. I will remove it.